src/Language/PureScript/Errors.hs

```diff
@@ -556,6 +558,16 @@ prettyPrintSingleError full e = prettyPrintErrorMessage <$> onTypesInErrorMessag
             ]
     goSimple (WildcardInferredType ty) =
       line $ "The wildcard type definition has the inferred type " ++ prettyPrintType ty
+    goSimple (NotExhaustivePattern bs) =
+      paras $ [ line "Pattern Match(es) are not exhaustive"
```

Comment:
Actually, what about `Patterns could not be determined to cover all cases`? That way, it's slightly clearer that they _might_ be exhaustive, but that we couldn't prove it.